config/Dockerfiles/JenkinsfileContainerBuild

```diff
@@ -163,8 +161,13 @@ podTemplate(name: podName,
                                 script: "pwd",
                                 returnStdout: true
                             ).trim()
-                            sh "buildah run -v ${cwd}/docs/source/examples/workspace:/workdir ${containerid} -- linchpin -c /etc/linchpin.conf -vvvp PinFile.dummy.yml -w /workdir up dummy-topo"
-
+                            sh "buildah copy --chown 10001:0 ${containerid} ${cwd}/config/Dockerfiles/linchpin/bin/run-linchpin /linchpin/bin/run"
+                            sh "buildah commit ${containerid} linchpin-test"
+                            string testcontainerid = sh (
+                                script: "buildah from linchpin-test",
+                                returnStdout: true
+                            ).trim()
+                            sh "buildah run -v ${cwd}/docs/source/examples/workspace:/workdir ${testcontainerid} uid_entrypoint run"
```

Comment:
this doesn't see, right to me..  The whole point of an entrypoint is that you don't have to specify it.  you should only have to specify run piece

Reply:
Again, this is only for testing. In a proper use of the item, you'd have docker exec (or run it in openshift) and this wouldn't be an issue.

Reply:
@p3ck This, I think, is mimicking the approach used in the existing docker file. In this scenario, the entrypoint is set to uid_entrypoint to handle adding the nonroot user to /etc/passwd, etc. That script the runs whatever else is specified in the CMD element. "run" is an arbitrary script that keeps the docker running if no other command is specified when executing the container.